modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/QuestionsBadge.es.js

```diff
@@ -13,18 +13,38 @@
  */
 
 import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
 import React from 'react';
 
-export default function QuestionsBadge({className, symbol, tooltip, value}) {
+export default function QuestionsBadge({
+	className,
+	iconInvert,
+	isActivityBadge,
+	symbol,
+	tooltip,
+	value,
+}) {
 	return (
 		<div
-			className={`c-py-2 c-px-3 rounded stretched-link-layer ${className}`}
+			className={classNames(
+				` c-py-2 c-px-3 rounded stretched-link-layer ${className}`,
+				{'badge-activity': isActivityBadge}
+			)}
 			data-tooltip-align="top"
 			title={tooltip}
 		>
-			<ClayIcon symbol={symbol} />
+			{symbol && <ClayIcon className={`${iconInvert}`} symbol={symbol} />}
```

Comment:
Simplify,

this iconInvert is weird, if it should be applied to a className, we should have a different name.

maybe symbolClassName


```suggestion
			{symbol && <ClayIcon className={iconInvert} symbol={symbol} />}
```